validation/src/test/scala/hmda/validation/rules/lar/validity/V545Spec.scala

```diff
@@ -0,0 +1,36 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V545Spec extends LarEditCheckSpec with BadValueUtils {
+
+  property("Valid when HOEPA status is 2") {
+    forAll(larGen) { lar =>
+      whenever(lar.hoepaStatus == 2) {
+        lar.mustPass
+      }
+    }
+  }
+
+  property("Valid when Lien status != 3") {
+    forAll(larGen) { lar =>
+      whenever(lar.lienStatus != 3) {
+        lar.mustPass
+      }
+    }
+  }
+
+  val invalidHoepaGen: Gen[Int] = Gen.choose(Int.MinValue, Int.MaxValue).filter(_ != 2)
```

Comment:
I don't think that it will return None unless the conditions are very specific or impossible (see [this documentation](https://github.com/rickynils/scalacheck/blob/master/doc/UserGuide.md)).  The docs aren't completely clear, but that seems to be the case.
If we still think this is a danger, we can always just use `retryUntil(some condition)`, which will try the generator again until it passes the condition